src/order/filter/filter_product.lean

```diff
@@ -11,13 +11,13 @@ universes u v
 variables {α : Type u} (β : Type v) (φ : filter α)
 local attribute [instance] classical.prop_decidable
 
-namespace filter 
+namespace filter
```

Comment:
Let's try to avoid making actual changes mixed in with whitespace changes to hundreds of lines.

Reply:
I think the problem is less my editor fixing those line endings than the people who keep committing those

Reply:
I don't disagree with fixing the line endings, but they make actual changes hard to notice.

Reply:
@rwbarton You can switch of whitespace-changes in your diff view.